include/nbl/asset/utils/CCPUMeshPackerV1.h

```diff
@@ -118,14 +73,14 @@ class CCPUMeshPackerV1 final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
 
 	~CCPUMeshPackerV1()
 	{
-		_NBL_ALIGNED_FREE(m_perInsVtxBuffAlctrResSpc);
+		if(isInstancingEnabled)
+			_NBL_ALIGNED_FREE(m_perInsVtxBuffAlctrResSpc);
 	}
 
 	template <typename MeshBufferIterator>
 	ReservedAllocationMeshBuffers alloc(const MeshBufferIterator mbBegin, const MeshBufferIterator mbEnd);
 
-	//TODO: test
-	void free(const ReservedAllocationMeshBuffers& ramb)  // TODO: this isn't special so do it in the base class
+	void free(const ReservedAllocationMeshBuffers& ramb)
```

Comment:
huh why not?

Reply:
`CCPUMeshPackerV1::free` differ from `CCPUMeshPackerV2::free`, also `CCPUMeshPackerV1::ReservedAllocationMeshBuffers` and `CCPUMeshPackerV2::ReservedAllocationMeshBuffers` are different types
Shared code could be somehow moved to the base class, but it would be complex and unnecessary imo

Reply:
did it myself.